charts/rancher-logging/0.2.4/charts/fluentd/charts/fluentd-linux/values.yaml

```diff
@@ -51,6 +51,6 @@ configmapReload:
   name: reloader
   image:
     repository: rancher/configmap-reload
```

Comment:
shouldn't line 6 above swap to 0.24 ?

Reply:
huh. Don't know how this happened. Fixed.